_includes/list_of_educators.html

```diff
@@ -38,15 +38,21 @@ <h3><a href="{{ person.url }}">{{ person.title }}</a></h3>
 </div>
 
 {% if column == 0 %}
-</div>
+</div> 
 {% endif %}
 
 {% endfor %}
 
 {% unless column == 0 %}
 <!-- The last </div> never happened, because we ended short of a full last column. -->
 <!-- Small bug: If there is no educator for a whole year, then this will still get displayed.... -->
+
+{% assign newyear = newyear % }
```

Comment:
I don't understand where this is defined... This statement will not do anything, right?

Reply:
Yes sir :(
@klieret  Sir basically I know c/c++ , python , html ,css and familliar with Machine learning so kindly guide me for a good contribution :)

Reply:
Thank you for your interesting in contributing. Unfortunately I currently do not have the time for detailed guidance. I'm not currently involved in GSOC, but good luck with your application :) 

I will close this PR since it does not solve the issue. Sorry about that!